integration/setup/tenant_cluster.go

```diff
@@ -130,15 +130,15 @@ func CRNotExistsCondition(ctx context.Context, id string, config Config) release
 	}
 }
 
-func ensureHostVPC(ctx context.Context, config Config) (string, error) {
-	stackName := "host-peer-" + env.ClusterID()
+func ensureHostVPC(ctx context.Context, id string, config Config) (string, error) {
```

Comment:
So it isn't possible to reuse the VPC?

Reply:
@calvix said this is a bad idea. So I wanted to test it that way. There are apparently resources being reused which we do not expect in the operator.

Reply:
I'd like to understand why this is a bad idea. We are not talking about reusing VPC between tests. We are talking about reusing VPC by clusters in a single test.

I'm fine with using separate VPC for them. But this will cause VPC explosion. And we have limits there.

Also I thought when we reuse the host VPC it's more production-like situation.

Reply:
Nothing will explode. We create one host peer VPC stack per e2e build already. What we do now for the IPAM test is that we not have one for the three clusters we create there within this specific e2e test, but for each of the three clusters one.

Reply:
Which for latest and previous is 4 additional VPCs. I don't want to demonize it. But still I don't understand why we shouldn't reuse it. Is that not working at all and would require additional work? Or just @calvix said it's a bad idea?

Reply:
It is not working at all for the IPAM test. 

<img width="1920" alt="screen shot 2018-11-07 at 17 53 18" src="https://user-images.githubusercontent.com/552769/48201161-b0106f00-e361-11e8-8cd2-8562719cb96c.png">

Vaclav confirmed that this is a bad idea. I want to try it the other way as proposed here.

Reply:
I talked with @calvix. I think we just need to have different values for those CIDRs between the clusters and it should work with single VPC: https://github.com/giantswarm/apiextensions/blob/befa4980fcfb7bf19596f088d3ee14de811451e6/helm/apiextensions-aws-config-e2e-chart/values.yaml#L22-L23

Reply:
I'm bit stubborn here. The reason is I feel this will in general make setup code more complex and scattered when using `defer`. I'm not blocking this though.

Reply:
Yes value from your screenshot matches with what we have there by default https://github.com/giantswarm/e2etemplates/blob/3aa5db40ae4484507d28cb2ec6992889a506d86e/pkg/chartvalues/apiextensions_aws_config_e2e_template.go#L10